tests/discontinuous_composition_bound_preserving_limiter/screen-output

```diff
@@ -9,8 +9,8 @@ Number of degrees of freedom: 8,164 (2,178+289+1,089+2,304+2,304)
    Rebuilding Stokes preconditioner...
    Solving Stokes system... 30+1 iterations.
 
-Number of active cells: 157 (on 6 levels)
-Number of degrees of freedom: 5,244 (1,482+195+741+1,413+1,413)
+Number of active cells: 151 (on 6 levels)
+Number of degrees of freedom: 5,035 (1,420+187+710+1,359+1,359)
```

Comment:
where does this difference come from? Shouldn't this patch make no difference to existing functionality?